kcidb/test_monitor.py

```diff
@@ -303,5 +303,5 @@ def test_mark_brown(self):
                     "Mark Brown <broonie@kernel.org>" in n.message.to
         ])
         self.assertEqual(len(subjects), 2)
-        self.assertRegex(subjects[0], r"^Builds failed for ")
-        self.assertRegex(subjects[1], r"^Tests failed for ")
+        self.assertRegex(subjects[0], r"^Testing done for ")
+        self.assertRegex(subjects[1], r"^Testing done for ")
```

Comment:
Would be good to check that the due time is set correctly here. However, for that to work reliably we would need to support passing the "current time" through the monitor.match() down to subscriptions. If you can figure out how to do that quickly, that would be great. If not, we can add it later.

Reply:
> 

@spbnick 
Yes I have figured out the problem with due 
I am checking if due is None and assigning timestamp to it 
This is the update: https://github.com/kernelci/kcidb/pull/270/commits/5100a415bf646e4d2ec88ca000ce96479df76b4c#diff-cb11dfc89a15f32781b56c749d3b12fc5292e76f816d6575ef91a4c1ca6d52d7R113

Reply:
Ah, nice! Thank you :)

Reply:
Here, however, I'm talking about adding two extra checks to the test to verify that the subscription is specifying the `due` argument to the constructor of the notification. It's not a big deal if we don't have it here now.